feature/gribi/ate_tests/gribigo_compliance_test/gribigo_compliance_test.go

```diff
@@ -95,13 +97,32 @@ func shouldSkip(tt *compliance.TestSpec) string {
 		return "This RequiresNonDefaultNINHG test is skipped by --skip_non_default_ni_nhg"
 	case *skipIdempotentDelete && tt.In.RequiresIdempotentDelete:
 		return "This RequiresIdempotentDelete test is skipped by --skip_idempotent_delete"
+	case *skipMPLS && tt.In.RequiresMPLS:
+		return "This RequiresMPLS test is skipped by --skip_mpls"
 	}
 	return moreSkipReasons[tt.In.ShortName]
 }
 
+func syncElectionID(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
+	clientA := &gribi.Client{
+		DUT:         dut,
+		Persistence: true,
+	}
+	t.Log("Establish gRIBI client connection with PERSISTENCE set to True")
+	if err := clientA.Start(t); err != nil {
+		t.Fatalf("gRIBI Connection for clientA could not be established")
+	}
+	clientA.BecomeLeader(t)
```

Comment:
You don't need to become a leader to learn the current election ID, right? (In fact, `BecomeLeader()` also calls `LearnElectionID` to understand the base).

So, seems like this `clientA.BecomelLeader` is not needed?

Reply:
thanks for pointing this out, fixed.